OpenRA.Mods.Common/Traits/BotModules/BuildingRepairBotModule.cs

```diff
@@ -35,7 +35,7 @@ void IBotRespondToAttack.RespondToAttack(IBot bot, Actor self, AttackInfo e)
 			var rb = self.TraitOrDefault<RepairableBuilding>();
 			if (rb != null)
 			{
-				if (e.DamageState > DamageState.Light && e.PreviousDamageState <= DamageState.Light && !rb.RepairActive)
+				if (e.DamageState > DamageState.Undamaged && e.PreviousDamageState <= DamageState.Undamaged && !rb.RepairActive)
```

Comment:
This is going to make the AI obsess over keeping its building health topped up... is that right?